### PR TITLE
add unix signals handler

### PR DESCRIPTION
**Pull request by kauanslr**

### CODE REVIEW

File: tests/Integration/Consumer/ConsumerTest.php [hunk 1]
```diff
@@ -68,6 +68,25 @@ public function test_it_should_throw_timeout_with_multiple()
         $consumer->callback($callback)->consume(1);
     }
 
+    /**
+     * @requires extension pcntl
+     * @
+     */
+    public function test_it_should_handle_sigterm_signal()
```
Comment (jleonardolemos) [inline tests/Integration/Consumer/ConsumerTest.php]: `signalHandler` can deal with three kinds of signals but just sigterm is beeing tested.

Comment (kauanslr) [reply tests/Integration/Consumer/ConsumerTest.php]: Yaeh, i've dont made it cause `signalHandler` has a simple switch so i dont want to test the switch itself, only the handling of signal

**Comment by kauanslr** [comment]
> The quit method and quitHard are the same, is it the desired behavior?
> Are they equals only for RabbitMQ Driver?

Yes, it's the desired behavour cause every driver can deal with diferent approachs when stoping the job

**Comment by kauanslr** [comment]
resolves #26

**Pull request merged**